python/cudf/cudf/core/tools/datetimes.py

```diff
@@ -221,8 +224,13 @@ def to_datetime(
                 format=format,
             )
             return as_index(col, name=arg.name)
-        elif isinstance(arg, cudf.Series):
-            col = arg._column
+        elif isinstance(arg, (cudf.Series, pd.Series)):
+            if isinstance(arg, (pd.Series)):
+                # this needs to work for a pandas series
+                # input or else it becomes an index
+                col = column.as_column(arg)
+            else:
+                col = arg._column
```

Comment:
I think we've abstracted away the need for this second level of branching into `as_column` already. Might be nice to simplify the code a little if so. Based on the following, I think `as_column` is a no-op if it receives a cuDF series.

https://github.com/rapidsai/cudf/blob/f08d6f1dc5c9c3e85b259b7091e3a86368bb87aa/python/cudf/cudf/core/column/column.py#L1723-L1726

Reply:
Ah, yes! I see what you mean. I'll change this.